src/wayland/seat/pointer/mod.rs

```diff
@@ -67,12 +67,11 @@ impl<D> PointerInternal<D> {
         }
     }
 
-    fn set_grab<G: PointerGrab<D> + 'static>(&mut self, serial: Serial, grab: G, time: u32) {
+    fn set_grab<G: PointerGrab<D> + 'static>(&mut self, serial: Serial, grab: G, unfocus: bool) {
```

Comment:
I wonder if, in terms of best-practices, it would not be better to replace this `bool` argument with an enum? Something like:

```rust
pub enum Focus {
    Keep,
    Clear
}
```

This way at call site it would be much easier to see at a glance what that argument means.

Reply:
Yeah, this is definitely more expressive, changed `set_grab` to take a `Focus` enum.